src/core/ColorPicker/Color.tsx

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import { getWindow } from '../utils/common';
+import { CommonProps } from '../utils/props';
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/color-picker.css';
+
+export type ColorProps = {
+  /**
+   * Color
+   */
+  color: string;
+  /**
+   * Handler for clicking a color
+   */
+  onColorClicked?: () => void;
+  /**
+   * Is color selected
+   */
+  isActive?: boolean;
+  /**
+   * User friendly color name
+   */
+  name?: string;
+  /**
+   * User Count Icon props.
+   */
+  tooltipRefProp?: React.ComponentPropsWithRef<'span'>;
+} & CommonProps;
+
+export const Color = (props: ColorProps) => {
+  const {
+    color,
+    style,
+    onColorClicked,
+    isActive,
+    tooltipRefProp,
+    ...rest
+  } = props;
+
+  useTheme();
+
+  const _style =
+    color && getWindow()?.CSS?.supports?.(`--swatch-color: ${color}`)
+      ? { '--swatch-color': color, ...style }
+      : { backgroundColor: color, ...style };
+
+  return (
+    <span
+      className={cx('iui-color-swatch', { [`iui-active`]: isActive })}
+      style={_style}
+      onClick={onColorClicked}
+      tabIndex={0}
```

Comment:
Not sure about this.. I guess expectation is, that you can move between colors with arrow keys.. Usually tab is used to focus whole element, but inner switch arrow keys could be better. 
So `Color` component would get tabIndex 0 only when its active and you will need to implement keyboard navigation yourself in ColorPicker...
There is example in DatePicker to check.
@bentleyvk do you feel tabing between colors is better than arrow keys?

Reply:
I agree that arrow navigation here is better because otherwise you would need to press tab e.g. for 20 times to go out.

Reply:
I added arrow key navigation